connectors/es/client.py

```diff
@@ -125,5 +125,7 @@ async def check_exists(self, indices=None, pipelines=None):
             except NotFoundError:
                 raise PreflightCheckError(f"Could not find pipeline {pipeline}")
 
-    async def delete_indices(self, indices=[]):
+    async def delete_indices(self, indices=None):
+        if indices is None:
+            indices = []
```

Comment:
Should we return directly?

Reply:
Yep we could, sorry already merged it. You can adjust if you want.

Reply:
No, it shouldn't. Will have a follow up to 1) make it mandatory, 2) not call ES if it's empty.